core_subnet_test.go

```diff
@@ -65,8 +65,9 @@ func (s *ResourceCoreSubnetTestSuite) SetupTest() {
 		ID:                 "id",
 		RouteTableID:       "routetableid",
 		SecurityListIDs: []string{
-			"slid1",
+			// Note: sorted by schema.HashString
```

Comment:
This test should verify that `HashString` is being used to sort the values, not the ordering of the items in the config.